src/strings/en-us.json

```diff
@@ -662,7 +662,7 @@
     "LabelEnableIP6": "Enable IPv6",
     "LabelEnableIP6Help": "Enable IPv6 functionality.",
     "LabelEnableLUFSScan": "Enable LUFS scan",
-    "LabelEnableLUFSScanHelp": "Enable LUFS scan for music (This will take longer and more resources).",
+    "LabelEnableLUFSScanHelp": "Enable LUFS data to be created during scanning of music. Useful for audio normalization. This will make the scan longer and take more resources.",
```

Comment:
I think the term "LUFS" might be the most confusing part of this feature for users not familiar with audio. How about something like this:

```
"LabelEnableLUFSScan": "Enable audio loudness scanning",
"LabelEnableLUFSScanHelp": "Clients can normalize audio playback to get equal loudness across tracks. This will make the scan longer and take more resources.",
```

The fact that Jellyfin uses LUFS internally doesn't matter to the end-user.

Reply:
I agree with you. Feel free to make any changes to the wording, I'll be happy if it'll make it clearer to the user :+1: